_episodes/01-introduction.md

```diff
@@ -64,13 +66,14 @@ Unsurprisingly, software installation and configuration challenges can have
 negative consequences for research: 
 - you can't use a specific tool at all, because it's not available or installable. 
 - you can't reproduce your results because you're not sure what tools you're actually using. 
-- you can't access extra resources because you're not able to replicate your software set up. 
+- you can't access extra/newer resources because you're not able to replicate your software set up. 
+- others cannot validate and/or build upon your work because they cannot recreate your system's unicorn configuration.
 
 Thankfully there are ways to get underneath (a lot of) this mess: containers 
 to the rescue! Containers provide a way to package up software dependencies 
 and access to resources such as files and communications networks in a uniform manner.
 
-### What is a Container? 
+### Docker and What is a Container? 
```

Comment:
The original idea was to introduce containers in abstract, which is why Docker isn't in the title... although then having "Docker" start the first sentence just adds cognitive load. How about "What is a Container? What is Docker?" maybe?